apps/site/.cloudflare/prepare-build.mjs

```diff
@@ -0,0 +1,57 @@
+//@ts-check
```

Comment:
?

Reply:
this is for getting some type checking in the file, anyways let me do it the appropriate way by including this file in the app's tsconfig

as part of this I can also probably improve the type checking of the other files in `.cloudflare` 👍

Reply:
I'll probably refrain from addressing this for now as I first should look into your request to move code out of this repo (which would change things here)